pkgs/development/python-modules/greynoise/0001-drop-ipaddress-from-requirements.patch

```diff
@@ -0,0 +1,37 @@
+From f2ef61e2559247f48a7ef52cd73969f2426cca69 Mon Sep 17 00:00:00 2001
+From: Maksym Balatsko <mbalatsko@gmail.com>
+Date: Sat, 30 Sep 2023 15:01:26 +0000
+Subject: [PATCH] drop ipaddress from requirements
+
+---
+ requirements/common.txt | 1 -
+ setup.py                | 1 -
+ 2 files changed, 2 deletions(-)
+
+diff --git a/requirements/common.txt b/requirements/common.txt
+index 8b3ba21..d745e30 100644
+--- a/requirements/common.txt
++++ b/requirements/common.txt
+@@ -5,7 +5,6 @@ colorama==0.4.6
+ click-default-group==1.2.2
+ click-repl==0.2.0
+ dict2xml==1.7.3;python_version>='3'
+-ipaddress==1.0.23
```

Comment:
You might be able to jailbreak this instead but let's see what happens upstream. We can drop this when we get that patch.